recipes/automated-testing.md

```diff
@@ -86,10 +86,26 @@ changes, we can make minimal changes to the test suite and/or mocked data.
 
 ### Integration Tests
 
-With these tests, we test the application API endpoints and assert that they are
-actually working as expected.
+With these tests, we test the combined behavior of multiple components within the system.
```

Comment:
This phrasing might be more intuitive (behaving together part, compared to combined behavior).
I might be subjective and wrong, so feel free to resolve if it doesn't make sense. :)
```suggestion
With these tests, we test how multiple components of the system behave together.
```